### PR TITLE
Backend PyTorch supports forward-mode automatic differentiation: Step 2

### PR DESCRIPTION
**Pull request by ZongrenZou**

### CODE REVIEW

File: deepxde/gradients/gradients_forward.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 __all__ = ["hessian", "jacobian"]
 
 from .jacobian import Jacobian, Jacobians
-from ..backend import backend_name, jax
+from ..backend import backend_name, torch, jax
```
Comment (lululxvi) [inline deepxde/gradients/gradients_forward.py:6]: `from ..backend import backend_name, jax, torch`

File: deepxde/gradients/jacobian.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from abc import ABC, abstractmethod
 
 from ..backend import backend_name
+from .. import config
```
Comment (lululxvi) [inline deepxde/gradients/jacobian.py:4]:
```
from .. import config
from ..backend import backend_name
```

File: deepxde/gradients/gradients_forward.py (hunk 1 above)
Comment (ZongrenZou) [reply deepxde/gradients/gradients_forward.py]: Done.

File: deepxde/gradients/jacobian.py (hunk 2 above)
Comment (ZongrenZou) [reply deepxde/gradients/jacobian.py:3]: Done.

**Pull request merged**